### PR TITLE
30 seeds

### LINKED ISSUE #30: Create working seed data.
Have a working set of seeds so that we can build features.

### PR DESCRIPTION
**Pull request by kevinprince**
Closes #30 

We can now seed hamster with some usable data which is good for playing with.

### CODE REVIEW

File: app/models/attendee.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Attendee < ActiveRecord::Base
-  attr_accessible :first_name, :last_name, :twitter, :address, :tshirt
```
Comment (mseckington) [inline app/models/attendee.rb:2]: So why does address not belong to an Attendee?

**Comment by mseckington** [comment]
:ok: for the rest

File: app/models/attendee.rb (hunk 1 above)
Comment (kevinprince) [inline app/models/attendee.rb:2]: :address is part of the email email now which attendee relates too.

Comment (mseckington) [inline app/models/attendee.rb:2]: Ah okay.

**Pull request merged**